### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.graphrbac.models.GraphErrorException
  fullName: azure.graphrbac.models.GraphErrorException
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - inheritance:
          - type: builtins.object
          type: builtins.BaseException
        type: builtins.Exception
      type: msrest.exceptions.ClientException
    type: msrest.exceptions.HttpOperationError
  langs:
  - python
  module: azure.graphrbac.models
  name: GraphErrorException
  source:
    id: GraphErrorException
    path: azure-graphrbac\azure\graphrbac\models\graph_error.py
    remote:
      branch: master
      path: azure-graphrbac\azure\graphrbac\models\graph_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 36
  summary: 'Server responsed with exception of type: ''GraphError''.

    '
  syntax:
    content: GraphErrorException(deserialize, response, *args)
    parameters:
    - description: 'A deserializer

        '
      id: deserialize
    - description: 'Server response to be deserialized.

        '
      id: response
  type: class
  uid: azure.graphrbac.models.GraphErrorException
references: []
